src/main/java/card/validation/CardInputArgumentsControl.java

```diff
@@ -0,0 +1,33 @@
+package main.java.card.validation;
+
+import java.io.IOException;
+
+public class CardInputArgumentsControl {
+
+    public CardInputArgumentsControl(String[] inputArgs, int expectedArgsNmb){
+        expectedArgumentsNmb = expectedArgsNmb;
+        inputArguments = inputArgs;
+    }
+
+    public void checkInput() throws IOException{
+        if(!isInputLentgthCorrect())
+            throw new IOException("Wrong number of input arguments.");
+
+        if(!isCardNumberCorrect())
+            throw new IOException("Didn't provide a card number in a proper format.");
+    }
+
+    private boolean isInputLentgthCorrect(){
```

Comment:
This if/else statement can be simplified, please correct typo in method name (Len**t**ght)  
```suggestion
    private boolean isInputLengthCorrect(){
        return inputArguments.length == expectedArgumentsNmb;
    }
```